test/main.cpp

```diff
@@ -1,4 +1,109 @@
-#include "ostream"
+#include "coordinate_3D_c.hpp"
+#include <iostream>
 
 #define CATCH_CONFIG_MAIN
 #include <catch.hpp>
+
+#include <iostream>
+
+TEST_CASE("Default constructor") {
+    r2d2::robot_arm::coordinate_3D_c c;
+    REQUIRE(c == r2d2::robot_arm::coordinate_3D_c(0, 0, 0));
```

Comment:
Are you sure this test is working correctly without a test for the == operator?

Reply:
Test passed. Please check latest commit.

Reply:
Ill accept it for now but maybe a test that checks if it is false should be added as well